src/main/java/com/futurcollars/lesson11/zad6/ResultWorkout.java

```diff
@@ -0,0 +1,30 @@
+package com.futurcollars.lesson11.zad6;
+
+public class ResultWorkout extends Workout {
```

Comment:
To jest typowy przypadek gdy używasz dziedziczenia nie ze względu na modelowanie tylko potrzebujesz jakichś metod. Spróbuj użyć tutaj kompozycji zamiast dziedziczenia :)